src/main/java/DBAccesser/User/QueryUser.java

```diff
@@ -0,0 +1,57 @@
+package DBAccesser.User;
+
+import Entity.Class.User;
+
+import org.springframework.stereotype.Component;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.cloud.gcp.data.spanner.core.SpannerTemplate;
+
+import com.google.cloud.spanner.Statement;
+import org.springframework.cloud.gcp.data.spanner.core.SpannerQueryOptions;
+
+import java.util.List;
+
+@Component
+public class QueryUser {
```

Comment:
Similar to what I commented on @simranmalik-4 's PR, let's merge this class and the class `InsertUser` into a class called `Users`.

Reply:
Update: The name of the merged class is `UserAccessor` and similarly for `Chat` and `Message`